src/Wpf.Ui/Controls/SplitButton/SplitButton.cs

```diff
@@ -85,29 +94,12 @@ public SplitButton()
         };
     }
 
-    protected virtual void AttachTemplateResources()
-    {
-        base.OnApplyTemplate();
-
-        if (GetTemplateChild(TemplateElementToggleButton) is ToggleButton toggleButton)
-        {
-            SplitButtonToggleButton = toggleButton;
-            AttachToggleButtonClick();
-        }
-        else
-        {
-            throw new NullReferenceException(
-                $"Element {nameof(TemplateElementToggleButton)} of type {typeof(ToggleButton)} not found in {typeof(SplitButton)}"
-            );
-        }
-    }
-
     private void AttachToggleButtonClick()
     {
         if (SplitButtonToggleButton != null)
         {
-            SplitButtonToggleButton.Click -= OnSplitButtonToggleButtonOnClick;
```

Comment:
This had to be changed to PreviewMouseLeftButtonUp for two reasons.
1) If the Click event is used, there's hardly any visible Pressed state as it instantly presses.
2) The SplitButton's context menu would immediately open on mouse down, this isn't the same in WinUI, where a mouse up is required.